spec/factories/courses.rb

```diff
@@ -93,6 +93,7 @@
 
     trait :with_accrediting_provider do
       association(:accrediting_provider, factory: :provider)
+      accrediting_provider_id { accrediting_provider.id }
```

Comment:
don't think that is needed

Reply:
Removed now the model uses the accrediting provider